app/models/backend/transaction.rb

```diff
@@ -174,29 +90,13 @@ def usage_accumulator
       @usage_accumulator ||= cinstance_data.usage_accumulator
     end
 
-    def save_to_storage
```

Comment:
These `save_to_storage` and `delete_from_storage` are really weird... 
What they used to do is: for **not** confirmed reports store them under key `transactions/{transaction_id}`, where the `transaction_id` is generated by incrementing the value in the `transactions/counter` key.

I think this is some old stuff, which we don't need. I don't think we should store anything in the Backend redis which is not what backend itself would store, even in test/development.